Automattic/newspack-plugin
# feat: add indentation to groups newsletters

### All Submissions:






### Changes proposed in this Pull Request:



Adds indentation to Group Subscription Lists in the Newsletters Wizard

See https://github.com/Automattic/newspack-newsletters/pull/1156

### How to test the changes in this Pull Request:

1. See the basic instructions in https://github.com/Automattic/newspack-newsletters/pull/1156
2. Visit Newspack > Engagement > Newsletters
3. Confirm that the Groups are nested under their Audiences

![image](https://user-images.githubusercontent.com/971483/234983909-3f7c18db-b037-41c7-af6d-b0e8607e5c63.png)


### Other information:
